docs/community/topics/i18n.rst

```diff
@@ -92,14 +104,36 @@ This section covers how to do so.
 
       pybabel init -i src/pydata_sphinx_theme/locale/sphinx.pot -d src/pydata_sphinx_theme/locale -D sphinx -l es
 
+   **To run this in ``.nox``**: ``nox -s translate -- init es``
+
 #. Edit the language's message catalog at ``pydata_sphinx_theme/locale/es/LC_MESSAGES/sphinx.po``. For each source string introduced by the ``msgid`` keyword, add its translation after the ``msgstr`` keyword.
 
 #. Compile the message catalogs of every language. This creates or updates the MO files with `PyBabel compile <https://babel.pocoo.org/en/latest/cmdline.html#compile>`__:
 
-.. code-block:: bash
+   .. code-block:: bash
+
+      pybabel compile -d src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static/locale -D sphinx
```

Comment:
This was `pybabel compile -d src/pydata_sphinx_theme/locale -D sphinx` - where is this new path from?

Reply:
ah whoops that was from when I was trying to get the `.mo` files to compile in the folder where all of our other compiled assets are located. But I backed out of that after deciding it was unnecessary complexity to add in this PR